loops-arrays-objects.js

```diff
@@ -99,20 +196,74 @@ const NOVIEmployees = [
 //  3581 = Oudwijk
 //  3583 = Schildersbuurt
 
-const students = [
-    {name: 'Mariska', city: 'Utrecht', zipCode: '3513', neighborhood: null},
-    {name: 'Mehmet', city: 'Utrecht', zipCode: '3514', neighborhood: null},
-    {name: 'Dennis', city: 'Utrecht', zipCode: '3572', neighborhood: null},
-    {name: 'Robin', city: 'Utrecht', zipCode: '3581', neighborhood: null},
-    {name: 'Tanush', city: 'Utrecht', zipCode: '3512', neighborhood: null},
-    {name: 'Florien', city: 'Utrecht', zipCode: '3513', neighborhood: null},
-    {name: 'Larissa', city: 'Utrecht', zipCode: '3583', neighborhood: null},
-    {name: 'Marijn', city: 'Utrecht', zipCode: '3572', neighborhood: null},
-    {name: 'Jan', city: 'Utrecht', zipCode: '3531', neighborhood: null},
-    {name: 'Laura', city: 'Utrecht', zipCode: '3531', neighborhood: null},
-    {name: 'Aicha', city: 'Utrecht', zipCode: '3514', neighborhood: null},
-    {name: 'Karima', city: 'Utrecht', zipCode: '3531', neighborhood: null},
-];
+
+const students = [{name: 'Mariska', city: 'Utrecht', zipCode: '3513', neighborhood: null}, {
+    name: 'Mehmet', city: 'Utrecht', zipCode: '3514', neighborhood: null
+}, {name: 'Dennis', city: 'Utrecht', zipCode: '3572', neighborhood: null}, {
+    name: 'Robin', city: 'Utrecht', zipCode: '3581', neighborhood: null
+}, {name: 'Tanush', city: 'Utrecht', zipCode: '3512', neighborhood: null}, {
+    name: 'Florien', city: 'Utrecht', zipCode: '3513', neighborhood: null
+}, {name: 'Larissa', city: 'Utrecht', zipCode: '3583', neighborhood: null}, {
+    name: 'Marijn', city: 'Utrecht', zipCode: '3572', neighborhood: null
+}, {name: 'Jan', city: 'Utrecht', zipCode: '3531', neighborhood: null}, {
+    name: 'Laura', city: 'Utrecht', zipCode: '3531', neighborhood: null
+}, {name: 'Aicha', city: 'Utrecht', zipCode: '3514', neighborhood: null}, {
+    name: 'Karima', city: 'Utrecht', zipCode: '3531', neighborhood: null
+},];
+
+// For loop schrijve
+for (let i = 0; i < students.length; i++) {
+    switch (true) {
+        case students[i].zipCode === '3513':
+            students[i].neighborhood = "Pijlsweerd";
+            break;
+
+        case students[i].zipCode === '3581':
+            students[i].neighborhood = "Oudwijk";
+            break;
+
+        case students[i].zipCode === '3514':
+            students[i].neighborhood = "Vogelenbuurt";
+            break;
+
+        case students[i].zipCode === '3512':
+            students[i].neighborhood = "Binnenstad";
+            break;
+
+        case students[i].zipCode === '3531':
+            students[i].neighborhood = "Lombok";
+            break;
+
+        case students[i].zipCode === '3572':
+            students[i].neighborhood = "Wittevrouwen";
+            break;
+
+        case students[i].zipCode === '3583':
+            students[i].neighborhood = "Schildersbuurt";
+            break;
+    }
```

Comment:
Zoals je het nu hebt gedaan lijkt het meer op een if-else. Waar een switch meer voor bedoeld is:
```suggestion
    switch (students[i].zipCode) {
        case '3513':
            students[i].neighborhood = "Pijlsweerd";
            break;

        case '3581':
            students[i].neighborhood = "Oudwijk";
            break;

        case '3514':
            students[i].neighborhood = "Vogelenbuurt";
            break;

        case '3512':
            students[i].neighborhood = "Binnenstad";
            break;

        case '3531':
            students[i].neighborhood = "Lombok";
            break;

        case '3572':
            students[i].neighborhood = "Wittevrouwen";
            break;

        case '3583':
            students[i].neighborhood = "Schildersbuurt";
            break;
    }
```